Solutions/Microsoft 365 Defender/Hunting Queries/Malicious Emails delivered.yaml

```diff
@@ -0,0 +1,13 @@
+id: cdac93ef-56c0-45bf-9e7f-9cbf0ad06456
+name: Malicious Emails delivered due to User/Admin Overrides
+description: |
+  This query helps surface phishing campaigns associated with Appspot abuse. These emails frequently contain phishing links that utilize the recipients' own email address as a unique identifier in the URI.
```

Comment:
Change the description according to the detection.

Reply:
Done